public/src/canadian-championship-bidding-policy.md

```diff
@@ -0,0 +1,73 @@
+<style type="text/css">
+  a[href]:after { content : "" }
+</style>
+
+Version: 1.0
+Updated: May 11, 2024
+
+# Canadian Championship Bidding Policy
+
+Uses of the words “must”, “must not”, “should”, “should not” and “may” match [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Speedcubing Canada’s Board of Directors reserves the right to modify this document if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
+
+Canadian Championships _should_ should be conducted every 2 years, unless otherwise noted by Speedcubing Canada's Board of Directors.
+
+## Canadian Championship Selection Committee Creation
+
+- For every Canadian Championship, one member of Speedcubing Canada's Board of Directors _should_ be selected to lead a separate Canadian Championship Selection Committee.
+- The Canadian Championship Selection Committee **must** include all current members of Speedcubing Canada's Board of Directors.
+- The selected leader _may_ add additional members as a part of the Canadian Championship Selection Committee.
+  - Selected members **must** be Speedcubing Canada members at the time of selection.
+  - Selected members _should_ have previous experience in organizing large competitions.
+
+## Canadian Championship Timeline Development
+
+- Submission dates for applications must be posted by the Canadian Championship Selection Committee at least 21 days before submissions are due.
+  - The application submission date _may_ be moved with written notice from Speedcubing Canada’s Board of Directors.
+- The Canadian Championship Selection period _should_ be a minimum of 14 days.
+- The selection of a host for the Canadian Championships _should_ be concluded at least 9 months before the start of the event.
+
+## Canadian Championship Application Submission Process
+
+- A Canadian Championship Application will consist of a PDF document that contains information to support the Canadian Championship Selection Committee with the Canadian Championship Selection Process.
+- Every application **must** have a point person if an application is being submitted on behalf of a larger organization team.
```

Comment:
```suggestion 
Applications that are being submitted on behalf of a larger organization team **must** have a point person.
```

Reply:
The document talks about the point person a lot, but this person only exists/is defined if the application is being submitted on behalf of a larger team. If an individual is submitting, then I assume all the requirements about the point person are intended to apply to that individual (such as "**must** be a member of Speedcubing Canada")?

Reply:
^ Was this noted? I'm still unclear.